src/game.c

```diff
@@ -251,6 +251,8 @@ int game_update(Game *game, float delta_time)
                 return -1;
             }
 
+            //TODO: Move this function somewhere else to allow volume adjustment in game
```

Comment:
Keep in mind that this will become a title of an issue: https://github.com/tsoding/snitch#todo-format

I'd suggest to add little bit of context to it. Something like "Move level picker volume update to a more appropriate place"